packages/eslint-plugin/dts/options.d.ts

```diff
@@ -22,7 +22,7 @@ export interface StylisticCustomizeOptions<Flat extends boolean = true> {
    *
    * @default 'single'
    */
-  quotes?: 'single' | 'double'
+  quotes?: 'single' | 'double' | 'backtick'
```

Comment:
Should we use `QuotesSchema0` directly?

Reply:
Maybe not, the point of this factory is to hide the implementation details of rules